src/view/store/cart/AddToCartInMemoryRepository.ts

```diff
@@ -1,15 +1,15 @@
 import { Product } from "../../../domain/Product";
+import { CartService } from '../../../services/cartService';
 import { AddToCartRepository } from "../../../useCases/addToCart";
 
-export default class AddToCartInMemoryRepository
-  implements AddToCartRepository {
-  private products: Product[] = [];
+export default class AddToCartInMemoryRepository implements AddToCartRepository {
```

Comment:
Since you inject `cartService` as an interface it seems like the name of this repository shouldn't mention any details of a concrete implementation of a service that you inject in it. You easily can inject the `HttpCartService` in this repository, so I would suggest giving this repository a more generic name.

Reply:
I find these names a bit weird too, but I believe the implementation of HttpCartService will have a slightly different implementation after all (since it's asynchronous, for example).

Reply:
I changed the names of the repository classes to adhere to the style the Adapter Pattern offers:

`AddToCartInMemoryRepository` -> `CartServiceToAddToCartRepository`
`ListProductsInMemoryRepository` -> `CartServiceToListProductsRepository`